lms/djangoapps/branding/api.py

```diff
@@ -261,7 +261,11 @@ def _footer_logo_img(is_secure):
         Absolute url to logo
     """
     logo_name = microsite.get_value('FOOTER_ORGANIZATION_IMAGE', settings.FOOTER_ORGANIZATION_IMAGE)
-    return _absolute_url_staticfile(is_secure, logo_name)
+    try:
+        return _absolute_url_staticfile(is_secure, logo_name)
+    except ValueError:
```

Comment:
The special thing about this case is that we are getting the footer logo URL from the microsite configuration, which falls back on the Django configuration, which loads it from the `lms.env.json` file, which is created by Ansible. In all other cases, the URL to the asset is set in the Mako template (or Django template), which allows it to be much more flexible.

Because of this cascade of information, we lose the ability to change the asset URL based on what theme is active. If you check the configuration repo, you'll see that [the footer URL is hardcoded to `themes/edx.org/images/logo.png`](https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml#L261), because that's the URL of the logo on edx.org's production site. This means that we lose out on all the flexibility of Django's staticfiles system, which seems like the worst of all possible worlds: tremendous complexity, without the benefit.

We also can't just default to the generic `images/logo.png` path in the configuration repo, because the Drupal site uses this URL as well, and doesn't have any logic for transforming it. As a result, it has to be a direct URL. Does that make sense?